### PR TITLE
Added example usage in the vignette file 

### PR DESCRIPTION
**Pull request by Birinder1469**
Added example usage in the vignette file 
doc 
Meta 
vignette folders created 
Two lines added in DESCRIPTION file

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -28,5 +28,6 @@ Imports:
     e1071,
     randomForest
 Suggests: 
-    testthat
+    testthat, knitr, rmarkdown
```
Comment (talhaadnan100) [inline DESCRIPTION:31]: Do `Suggests` packages need to be separated into a new line like `Imports`?

**Review by talhaadnan100 — COMMENTED** [review]
Great job getting this started. Is the `vignettes/caretcompaR.Rmd` complete or is this intended as a first draft?

File: DESCRIPTION (hunk 1 above)
Comment (Birinder1469) [reply DESCRIPTION:31]: They work well this way.

**Comment by Birinder1469** [comment]
Once all our functions are ready we can make vignette elaborate for now it has toy example of the two functions out of 3.

**Pull request merged**